docs/docs/explanations/architecture.md

```diff
@@ -8,61 +8,111 @@ sidebar_position: 2
-API Container Status:                 RUNNING
-API Container Host GRPC Port:         127.0.0.1:60768
-API Container Host GRPC Proxy Port:   127.0.0.1:60769
 
 ========================================== User Services ==========================================
 GUID                ID       Ports                             Status
 nginx-1669299161    nginx    http: 80/tcp -> 127.0.0.1:60785   RUNNING
 nginx2-1669299176   nginx2   http: 80/tcp -> 127.0.0.1:60794   RUNNING
 ```
 
+API Container
+-------------
+The second type of container that Kurtosis creates is the API container. One API container is created per enclave, and each API container is responsible for managing interactions with its own enclave. All manipulation of an enclave's contents happens via API calls to the enclave's API container. 
+
+Example: a Kurtosis API container running in Docker:
+
+```
+3c0b6ab7bb85   kurtosistech/core:0.70.7                          "/bin/sh -c ./api-co…"   20 hours ago    Up 20 hours    0.0.0.0:58419->7443/tcp, 0.0.0.0:58418->7444/tcp   kurtosis-api--6babc3090ad04184b2094901a7ead7b4
+```
+
+Starlark
+--------
+Distributed system definitions are complex. Therefore, there are many, many ways to instantiate, configure, and manipulate an enclave. To provide the required power, manipulations to an enclave are expressed using [Starlark][starlark-reference] scripts.
+
+To manipulate an enclave, users upload Starlark scripts to the API container. The API container executes the instructions in the script, and the enclave's contents will be mutated.
+
+Example: This Starlark snippet from [the quickstart][quickstart] launches a Postgres container:
+
+```python
+def run(plan, args):
+    postgres = plan.add_service(
+        service_name = "postgres",
+        config = ServiceConfig(
+            image = "postgres:15.2-alpine",
+            ports = {
+                "postgresql": PortSpec(5432, application_protocol = "postgresql"),
+            },
+            env_vars = {
+                "POSTGRES_PASSWORD": "password",
+            },
+        ),
+    )
+```
+
+For a list of all the possible operations on an enclave, [see this reference][starlark-instructions].
+
 SDKs
 ----
-All interaction with Kurtosis happens via API requests to the Kurtosis engine. To assist with calling the API, [we provide SDKs in various languages](https://github.com/kurtosis-tech/kurtosis-sdk). Anything the Kurtosis engine is capable of doing will be available via the API and, therefore, via the SDKs.
-
-For day-to-day operation, we also provide [a CLI ][installation] (usage guide [here][cli-usage]). This is a Go CLI that uses the Go SDK underneath.
+All interactions with Kurtosis happen through API requests to the Kurtosis containers. To assist with calling the API, [we provide SDKs in various languages](https://github.com/kurtosis-tech/kurtosis-sdk). Anything the Kurtosis can do will be available via the API and, therefore, via the SDKs.
```

Comment:
Was just browsing through the docs to verify! Lets change this to https://github.com/kurtosis-tech/kurtosis/tree/main/api

Reply:
Deleting this in favor of the suggestion